libr/include/rvc.h

```diff
@@ -18,6 +18,7 @@ R_API int r_vc_git_add(const char *path, const char *fname);
 R_API int r_vc_git_commit(const char *path, const char *message);
 
 R_API bool r_vc_commit(const char *rp, const char *message, const char *auth,
```

Comment:
that fits in one line, function signatures should never be wrapped in multilines for grepping purposes